build.gradle

```diff
@@ -13,8 +13,8 @@ plugins {
 group = 'uk.gov.hmcts.reform'
 version = '0.0.1'
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 1.11
+targetCompatibility = 1.11
```

Comment:
missed this one in previous review - the diff was incomplete. iirc, this should be just `11`, right?

Reply:
yup will do that after 1.9 Gradle docs says 11,12... but 1.11 also works :)

Reply:
hm, maybe they did a patch - used to not work 🤷‍♂